.github/ISSUE_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@ IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOU
 
 **I'm submitting a ...**
 <!-- (check one with "x") -->
-- [ ] regression from 1.7.0
+- [ ] regression from 1.8.0
```

Comment:
Shouldn’t it be “regression from 1.7.x/1.8.x”?

Reply:
I am not sure... I'll change it to your suggestion.

Reply:
Actually I am going to revert and leave this as 1.7.0.